v22.1/create-table.md

```diff
@@ -1,4 +1,4 @@
----
+PostgreSQL---
```

Comment:
I think this is a typo. It breaks the metadata.

Reply:
Oops. Fixed.